.travis.yml

```diff
@@ -20,12 +20,11 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
-        - NUMPY_VERSION=1.9
+        - NUMPY_VERSION=1.10
```

Comment:
One counterargument could be that one should add a test at the bottom against the (stable-1) version once there is a new release. Using ``stable`` here probably makes that update a tiny bit less likely.